.travis.yml

```
@@ -100,3 +100,9 @@ env:
     - secure: "O3pD+CI5Twbdu4qXwW0MAX9mAYTYMGmZWvUJe4yNO5n/O90SGsb8fVPWjtw6iBX3FZFyGary0VGvQFJtdYRO8xj8NqYG1teJWsMYzqobQ5CEgqKdcPoRfPTmS0m2RR5ThL8KagGMfWAuy+7+OCbbocuPxpDhGbQ53UjcmRFrQicsy/7YEM2woafh7s/uejAgEJL7v2HKeR82ZVUGcDXAmAF2Djyn9zmMddPRvkftCdcquCqbQocj162WOjKG/wDm1u+Amd7ZzIoop23Oxmc4p1HSNex7OOFwTsrA95K8H3mNEzO2NQ6fr458ghfjXAEqi3Sa47ZgyqmcQ7f+XSfk9v/vzehsZxtrm81GpYmYq3hg952Fh6eP2sQEC+LOaBcYOETHQoCxYN/vYX4Mqy9QCEbr6WjuIagjQzcC8ZjR6q0yCYPnEF8NDyPghp8+0UaLUv/Qd/9k/+6AX8OFIO+fVW5jawM4M2/NF96gH8GgSue7jocf4EoBCx+BNVY8ulpq/VWtCBifYVpXSPPMNJXr3pEoeRKEVFD3h5nLFNw6PKXL9hITRkz9G4LoWXSMX0NdJ6dDr3IGeNrrr1ZDNa0CCKRlCIT5bMh5dWE7YjAVwD8FvX9SRhGL+jrFOZyahnN4un6aprwC6vzkgqSeJsE2I/L31LlxV77mHQ+Z3dUeAxQ="
     # encrypt with: travis encrypt --pro RSYNC_PASSPHRASE=...
     - secure: "pHQSdAYLDBvvJV/26zNz9OAzKnfcUHiRrTdNigxK2bV6vwrtL1TPzBOdkL69oV84iPAFX3rZGmJflZ3xRC2PlUFSvXDflD2zjgXdZxiI0+TwUoqZSoAsjJkC4dIQRlt3nc3BhCC1LKFTF2shOBCH//RQUGiDTxgo6kEu++xsSwRd2rY4F0RhMGTcyDVXiTG/33bEjm4xxFGxwUsad4ao6km79szoCIg3h+m0nlKAlUWFOhi+I7Cj6M4Hdpk4THXTdM2Amn1Qrc2no1FkFc8w0HuyNxYZDi5vtXam8Bjx5YNY35Z3rQLwov1VMjS3sG95rfA44yp7N0eQ83eg7Qwe4ZACAfNcUT9z8IwpsdvdsMROQIDYc5KxvAugl/XiRcEL/UitI2KglZr2AwRE/A5eR68IzNOijeo2mP7BmUMFkGw5rTW7P+BDQ90j+Nv7bh2zllQ0tks4xHA5nI0QbJPyZ9MeRK8J81xN3iyIvoW0T8vOLL3TWfg8tmES0OJM1IveuQyVZssCWdxgKkv5nRIwEN7G6qACcGaDsQgZn9+J26HGQ6l2vKAAK28B5fEbcCphUeJvVyNwmUDQ3KyF/QgDCDRTcyPhnupX4jOa2+pw6HrK9e3LHOJCJUbLfc3ZwvwHQu2I+P3HCU/PM58R7aw14k2PltCiUSB1CxBL7S8Nx7c="
+
+# safelist
+branches:
+  only:
+    - master
+    - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
```

```suggestion
    - /^\d+\.\d+(\.\d+)?(-\S*)?$/
```

@ennru, since we don't have `release-` branches here, let's stay with the status-quo here. We can review it if needed in the future.